proxy/mempool/mempool_schedule.py

```diff
@@ -408,6 +414,11 @@ def add_tx(self, tx: MPTxRequest) -> MPTxSendResult:
         sender_pool = self._get_or_create_sender_pool(tx.sender_address)
         LOG.debug(f'Got sender pool {tx.chain_id, tx.sender_address} with {sender_pool.len_tx_nonce_queue} txs')
 
+        if self.tx_cnt >= self._capacity_high_watermark:
+            pending_nonce = 1 if sender_pool.is_empty() else sender_pool.pending_nonce
```

Comment:
This condition doesn't look good.
As a result, all txs for new users will be rejected, because they all won't have txs in the pool.

For the case, when send_pool is empty we should compare `tx.nonce` with `state_tx_cnt`.
We should allow to add tx if nonce is equal to the state_tx_cnt.
Please check the line 429 in this file.

Reply:
state_tx_cnt is calculated in neon_rpc_api_model/transaction_validator.py